Modules/Loadable/Markups/MRML/vtkCurveMeasurementsCalculator.h

```diff
@@ -66,11 +72,26 @@ class VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkCurveMeasurementsCalculator : pub
   /// Get name of max curvature measurement
   static const char* GetMaxCurvatureName() { return "curvature max"; };
 
-  vtkMTimeType GetMTime() override;
-
   vtkGetMacro(CurvatureUnits, std::string);
   vtkSetMacro(CurvatureUnits, std::string);
 
+  //@{
+  /// Set/Get flag determining whether to calculate torsion
+  vtkSetMacro(CalculateTorsion, bool);
+  vtkGetMacro(CalculateTorsion, bool);
+  vtkBooleanMacro(CalculateTorsion, bool);
+  //@}
+
+  /// Get name of average torsion measurement
+  static const char* GetAverageTorsionName() { return "torsion avg"; };
```

Comment:
We usually use `mean` instead of average, as it is shorter and maybe a bit more exact. We already use `mean` here for curvature, so let's rename these `GetMeanTorsionName` and `torsion mean`.

Reply:
Ah yes, this was from the algorithm of the first approach that I implemented. I'll rename it.